pkg/querier/querier.go

```diff
@@ -40,6 +43,19 @@ type AgentNetworkPolicyInfoQuerier interface {
 	GetRuleByFlowID(ruleFlowID uint32) *types.PolicyRule
 }
 
+type AgentMulticastInfoQuerier interface {
+	// GetIGMPReportNPStats gets statistics generated by networkpolicies that block or allow
+	// IGMP reports message. The statistics returned are incremental and will be reset after
+	// reporting to antrea-controller.
+	GetIGMPReportNPStats() (anpStats, acnpStats map[apitypes.UID]map[string]*types.RuleMetric)
```

Comment:
To be consistent, this func should be renamed to CollectIGMPReportNPStats() too. Maybe we can remove "Report" from the name, so CollectIGMPNPStats().

Reply:
updated